src/typegoose-core.module.ts

```diff
@@ -102,6 +103,8 @@ export class TypegooseCoreModule implements OnModuleDestroy {
 
     if (connection) {
       await connection.close();
+      models.clear();
```

Comment:
dont delete all models & constructors, try to find all models that are just for this connection, and delete them

Reply:
Yep, It's good idea too reset cache just from models which connection has closed.
But `deleteModel` will not help in this case https://github.com/typegoose/typegoose/blob/1fc037031fabf6c586aad5dc947328ae4c5da7ed/src/typegoose.ts#L225

It will try to delete model in mogoose just from first/main connection

Reply:
and in this typegoose version there is not such method. It has release in 6.1.0
https://github.com/typegoose/typegoose/commit/c6640198c5de14c6c48b1d9f060bdf4c0c2087e8

Reply:
I've fixed it and create PR https://github.com/typegoose/typegoose/pull/119
So we can update typegoose after release